include/cpp2util.h

```diff
@@ -364,7 +364,7 @@ constexpr auto gcc_clang_msvc_min_versions(
     auto msvc
 )
 {
-    return gcc_ver > gcc || clang_ver > clang || msvc_ver > msvc;
+    return gcc_ver >= gcc || clang_ver >= clang || msvc_ver >= msvc;
```

Comment:
After https://github.com/hsutter/cppfront/pull/1263#discussion_r1733509413 I've updated the code to accept the provided value to better fit the name (`gcc_clang_msvc_min_versions`).